cdap-proto/src/main/java/co/cask/cdap/proto/Id.java

```diff
@@ -27,6 +27,7 @@
 import co.cask.cdap.proto.id.EntityIdCompatible;
 import co.cask.cdap.proto.id.FlowletId;
 import co.cask.cdap.proto.id.FlowletQueueId;
+import co.cask.cdap.proto.id.InstanceId;
```

Comment:
yeah will revert. Had added it because the ``EntityId`` classes enforce that you have an old version of the class in ``Id`` as well. I relaxed that limitation (for newer Ids), but forgot to revert this change.